lib/roda/plugins/exception_page.rb

```diff
@@ -54,6 +54,13 @@ module RodaPlugins
     #   Roda::RodaPlugins::ExceptionPage.css
     #   Roda::RodaPlugins::ExceptionPage.js
     #
+    # To customize the CSS or JS that applies to the exception page without creating
+    # static files, you can specify the values during plugin configuration:
+    #
+    #   plugin :exception_page,
+    #     css: ExceptionPage.css.dup.prepend("html { background: '#f00'}"),
```

Comment:
Are you sure this works?  I would doubt it as the `ExceptionPage` constant is probably not in scope and not loaded at the time this is evaluated.

Reply:
Ah, yeah, I was trying to think of how you could augment the current CSS. I will revisit.